library/irssi

```diff
@@ -1,16 +1,16 @@
-# maintainer: Jessie Frazelle <jess@docker.com> (@jfrazelle)
+# maintainer: Jessie Frazelle <acidburn@google.com> (@jessfraz)
```

Comment:
Did you make this email address fix over in `generate-stackbrew-library.sh` too?  (I can if you didn't :heart:)